src/Raven.Server/Documents/Handlers/ReplicationHandler.cs

```diff
@@ -305,7 +305,7 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                     {
                         ["Key"] = new DynamicJsonValue
                         {
-                            [nameof(item.Key)] = item.Key.GetType(),
+                            [nameof(item.Key)] = item.Key.GetType().ToString(),
```

Comment:
That is the type, so why is it called `Key`.
Also, let's avoid adding type names to the protocol. Let's use an enum

Reply:
this isn't a protocol.. this a debug/studio EP
This also the reason why I stayed with the "Key" here.

Reply:
That is part of our endpoints, which is why I would rather have an enum over type name, which may change arbitrarily